src/helpers/analyticsEvents/highlighting/editNote.ts

```diff
@@ -0,0 +1,25 @@
+import { createSelector } from 'reselect';
+import * as selectNavigation from '../../../app/navigation/selectors';
+import { AnalyticsEvent } from '../event';
+
+const editNote = 'REX highlighting - edit highlight note';
+const addNote = 'REX highlighting - note added';
```

Comment:
to be consistent with the others
```suggestion
const addNote = 'REX highlighting - add note';
```